blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/io/astyanax/AstyanaxWriterIntegrationTest.java

```diff
@@ -16,87 +16,66 @@
 
 package com.rackspacecloud.blueflood.io.astyanax;
 
+import com.rackspacecloud.blueflood.io.CassandraModel;
 import com.rackspacecloud.blueflood.io.IntegrationTestBase;
-import com.rackspacecloud.blueflood.io.astyanax.AstyanaxWriter;
+import com.rackspacecloud.blueflood.io.TestIO;
 import com.rackspacecloud.blueflood.types.Locator;
 import org.junit.Test;
 
+import static org.junit.Assert.*;
+
+
 public class AstyanaxWriterIntegrationTest extends IntegrationTestBase {
 
+    private TestIO testIO = new AstyanaxTestIO();
+
     @Test
     public void testEnsureStringMetricsDoNotEndUpInNumericSpace() throws Exception {
-        assertNumberOfRows("metrics_string", 0);
-        assertNumberOfRows("metrics_full", 0);
-        assertNumberOfRows("metrics_locator", 0);
+
+        assertEquals( 0, testIO.getKeyCount( CassandraModel.CF_METRICS_STRING_NAME ) );
+        assertEquals( 0, testIO.getKeyCount( CassandraModel.CF_METRICS_FULL_NAME ) );
+        assertEquals( 0, testIO.getKeyCount( CassandraModel.CF_METRICS_LOCATOR_NAME ) );
 
         writeMetric("string_metric", "This is a string test");
-        
-        assertNumberOfRows("metrics_string", 1);
-        assertNumberOfRows("metrics_full", 0);
-        assertNumberOfRows("metrics_locator", 0);
+
+        assertEquals( 1, testIO.getKeyCount( CassandraModel.CF_METRICS_STRING_NAME ) );
+        assertEquals( 0, testIO.getKeyCount( CassandraModel.CF_METRICS_FULL_NAME ) );
+        assertEquals( 0, testIO.getKeyCount( CassandraModel.CF_METRICS_LOCATOR_NAME ) );
```

Comment:
if any of these asserts fail, it will be difficult to tell which one did.  If you write a comment as below, it can help.

`assertEquals( "Key count for METRICS_FULL", 0, testIO.getKeyCount( CassandraModel.CF_METRICS_FULL_NAME ) );`